source/manual/alerts/search-reindex-failed.html.md

```diff
@@ -0,0 +1,22 @@
+---
+owner_slack: "#2ndline"
+title: Search reindex failed
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2017-11-23
+review_in: 12 months
+---
+
+The reindex task is run weekly on a Monday at 12pm on integration.  This is to ensure the process works as
+expected when we need to run it in production.  This task is be manually run in production by the
```

Comment:
Typo: "is be". 🙃

Reply:
I hate documentation. this should be correct now (I hope)